frontend/src/components/pages/ComponentLibrary.tsx

```diff
@@ -1,15 +1,65 @@
 import React from "react";
+import { Box, Center } from "@chakra-ui/react";
 
-import StudentDashboardHeader from "../assessments/assessment-creation/StudentDashboardHeader";
+import { QuestionComponentResponse } from "../../APIClients/types/TestClientTypes";
+import { QuestionElementType } from "../../types/QuestionTypes";
+import DisplayQuestion from "../assessments/assessment-creation/DisplayQuestion";
+
+const MOCK_DATA: QuestionComponentResponse[] = [
+  {
+    type: QuestionElementType.QUESTION_TEXT,
+    metadata: {
+      questionText:
+        "Johnny is selling 19 apples at his store. Thomas buys 7 apples, Rick buys 2 apples, and Mike buys 3 apples. Then Thomas gives Rick 1 apple and Mike 3 apples. How much do they each have left now?",
+      __typename: "QuestionTextMetadata",
+    },
+  },
+  {
+    type: QuestionElementType.TEXT,
+    metadata: {
+      text: "How many apples can you eat?",
+      __typename: "TextMetadata",
+    },
+  },
+  {
+    type: QuestionElementType.SHORT_ANSWER,
+    metadata: {
+      answer: 5,
+      __typename: "ShortAnswerMetadata",
+    },
+  },
+  {
+    type: QuestionElementType.MULTIPLE_CHOICE,
+    metadata: {
+      options: ["First", "Second", "Third", "Fourth"],
+      answerIndex: 2,
+      __typename: "MultipleChoiceMetadata",
+    },
+  },
+  {
+    type: QuestionElementType.TEXT,
+    metadata: {
+      text: "How many pears can you eat?",
+      __typename: "TextMetadata",
+    },
+  },
+  {
+    type: QuestionElementType.MULTI_SELECT,
+    metadata: {
+      options: ["One", "Two", "Three", "Four", "Five"],
+      answerIndex: 0,
+      __typename: "MultiSelectMetadata",
+    },
+  },
+];
 
 const ComponentLibrary = (): React.ReactElement => {
   return (
-    <div>
-      <StudentDashboardHeader
-        assessmentName="Unit 0 Review Test"
-        classroomName="Mathematics 4 - Mr. Roberts"
-      />
-    </div>
+    <Center>
+      <Box w="60%">
+        <DisplayQuestion questionComponents={MOCK_DATA} />
+      </Box>
+    </Center>
   );
 };
```

Comment:
let's keep the old components in the `ComponentLibrary` when we add new ones! we usually try to keep things in the `ComponentLibrary` for reference on how to use them when we haven't used them in actual app pages. 

then, we remove when they exist in app pages (b/c we can just reference how to use them there!)

Reply:
ohhhh sorry about that